proton-c/bindings/python/proton/__init__.py

```diff
@@ -2581,6 +2581,9 @@ def close(self):
     """
     self._update_cond()
     pn_connection_close(self._impl)
+    if hasattr(self, '_session_policy'):
```

Comment:
Before my delta, the existing use of the attribute was via a hasattr() test.  Elsewhere in the code, hasattr attributes seemed to be decommissioned via del instead of None (more pythonic?). Functionally I have no preference between the two.